execution/src/windows/execution.cc

```diff
@@ -564,10 +565,10 @@ namespace leatherman { namespace execution {
         if (use_job_object) {
             hJob = scoped_handle(CreateJobObjectW(nullptr, nullptr));
             if (hJob == NULL) {
-                LOG_ERROR("failed to create job object: %1%.", windows::system_error());
+                LOG_ERROR("failed to create job object: {1}.", system_error());
```

Comment:
Why does this trip the `windows` namespace from `system_error`, but none below do?